### ISSUE TITLE
Allow selecting which apps have unlimited quota

### ISSUE THREAD

**Issue by stevesoltys** [stevesoltys/backup#554]
This makes things more configurable, especially if we end up with large D2D backups where we want to allow going over quota for certain apps but not others.

**Comment by grote** [comment]
We already have an expert option to turn off quota limits completely. Maybe for d2d backups we'd need this to force-on anyway since d2d probably assumes unlimited quota anyway.

**Comment by stevesoltys** [comment]
Yeah this ticket would be for selectively choosing which apps can exceed quota via settings (like we have for exclusions).

**Comment by grote** [comment]
>  large D2D backups where we want to allow going over quota for certain apps but not others.

did you encounter any use-case already in your personal collection of apps? Note that if an app goes over quota, it wouldn't get backed up at all.
